.storybook/main.js

```diff
@@ -1,15 +1,15 @@
 require('dotenv').config();
 
 module.exports = {
-  addons: [
-    '@storybook/addon-essentials',
-    '@storybook/addon-storysource',
-    'storybook-addon-performance/register',
-  ],
+  addons: ['@storybook/addon-essentials', '@storybook/addon-storysource'],
   check: true,
   checkOptions: {
     tsconfig: '../stories/tsconfig.json',
   },
+  disableTelemetry: true,
```

Comment:
I think this need to be moved under core e.g.

```suggestion
  core: {
    disableTelemetry: true,
  },
```